README.md

```diff
@@ -35,8 +35,6 @@ Alternatively, download manually:
 
  * [rrule.min.js](https://jakubroztocil.github.io/rrule/dist/es5/rrule.min.js) (bundled, minified)
  * [rrule.js](https://jakubroztocil.github.io/rrule/dist/es5/rrule.js) (bundled, not minified)
- * [rrule-tz.min.js](https://jakubroztocil.github.io/rrule/dist/es5/rrule-tz.min.js) (with timezone support, bundled, minified)
- * [rrule-tz.js](https://jakubroztocil.github.io/rrule/dist/es5/rrule-tz.js) (with timezone support, bundled, not minified)
```

Comment:
If there are projects loading from these github.io urls, for example using a <script> tag in an html file, this change will break those projects. I recommend we make a copy of rrule.min.js and rrule.js to rrule-tz.min.js and rrule-tz.js and continue to place them under dist folder.
But on the other hand, I don't know can we push users of these 2 files to switch. If we do not want to keep provide these files forever, eventually we need to stop and whenever we stop, it will break some body, so why not break it now.

Reply:
I hope nobody is linking directly to these files in a production app! It would be a very bad idea, since they're not versioned and can change unpredictably. Thus, I think it's a good idea to discourage that use case. These files are for direct download only. I would support removing even these links, and encourage everyone to just use npm as the source of truth for the package, but I think that's out of scope for this PR.